lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -1657,9 +1657,25 @@ class DxilRDATWriter : public DxilPartWriter {
     }
 
     if (props.Node.LaunchType == NodeLaunchType::Mesh) {
+      const DxilEntrySignature &sig = entryProps.sig;
       nAttrib = {};
       nAttrib.AttribKind = (uint32_t)RDAT::NodeFuncAttribKind::MeshShaderInfo;
       RDAT::MSInfo info = {};
+      uint32_t shaderFlags = 0;
+      info.SigOutputElements = AddSigElements(sig.OutputSignature, shaderFlags);
+      funcInfo.ShaderFlags |=
+          (uint16_t)(shaderFlags &
+                     (uint16_t)DxilShaderFlags::OutputPositionPresent);
+      info.SigPrimOutputElements =
+          AddSigElements(sig.PatchConstOrPrimSignature, shaderFlags);
+      // TODO: Fill in ViewID related masks
+      info.NumThreads =
+          Builder.InsertArray(&props.numThreads[0], &props.numThreads[0] + 3);
+      info.GroupSharedBytesUsed = tgsmSizeInBytes;
+      info.GroupSharedBytesDependentOnViewID =
+          (uint32_t)0; // TODO: same thing (note: this isn't filled in for PSV!)
```

Comment:
"same thing"?  Is this referring to line 1671?  Suggest making the TODO comment self-contained in case some future revision updates one without spotting the other.

Reply:
The mesh nodes spec identifies these as set to zero for the preview (ViewID not supported in preview).  We could group the ViewID related properties and have a comment that reflects this decision from the spec instead.